pegasus/routes/v2_user_routes.rb

```diff
@@ -3,7 +3,11 @@
   dont_cache
   forbidden! unless dashboard_user
   content_type :json
-  JSON.pretty_generate(dashboard_user.slice_keys(:id, :name))
+  result = dashboard_user.slice_keys(:id, :name, :admin)
+  result[:owned_sections] = DASHBOARD_DB[:sections].
```

Comment:
I'm not sure what you're getting at, since `slice_keys` is a `Hash` utility method with no special knowledge of the `dashboard_user`.  Would you prefer this whole detail be hidden away in a method on DashboardUser?